staker/block_validator.go

```diff
@@ -173,6 +179,7 @@ var TestBlockValidatorConfig = BlockValidatorConfig{
 	Enable:                   false,
 	ValidationServer:         rpcclient.TestClientConfig,
 	ValidationServerConfigs:  []rpcclient.ClientConfig{rpcclient.TestClientConfig},
+	ExecutionServerConfig:    rpcclient.TestClientConfig,
```

Comment:
need to also add ExecutionServerConfig to DefaultBlockValidatorConfig with same value as ValidationServer